src/error.rs

```diff
@@ -1,11 +1,14 @@
+#![allow(unused_doc_comment)]
```

Comment:
Can you just add doc comments like for the other error types so we don't need this?

Reply:
That warning is not for missing docs, it's for docs that are not being used, from what I know it's an issue with `error-chain`.

Reply:
Oh ok, yeah I've seen that one. I'll bump up the error-chain version too.